web-client/src/views/Messages/MessageList.tsx

```diff
@@ -0,0 +1,396 @@
+import {
```

Comment:
This is "the" message table. You pass in what columns and filters you want, and it takes care of the rest.

Eventually, I would love to see a message-independent "you pass in what columns, filters, and data you want (maybe with specific components for specific cells), and it takes care of the rest" class.